tests/common/pythonhashseed_test.py

```diff
@@ -0,0 +1,4 @@
+from deep_qa.common.checks import ensure_pythonhashseed_set
+
+def test_pythonhashseed():
+    ensure_pythonhashseed_set()
```

Comment:
I wonder if it's worth doing some "assert fails" thing here, and giving an error message specifically about other tests possibly failing because of this one.